README.md

```diff
@@ -155,17 +156,56 @@ The default is 70 characters for each of these:
 }
 ```
 
+### new-line-at-eof
```

Comment:
You duplicated this (same documentation exists in line 142). Do you mind deleting it?